scripts/verify-registry-install.sh

```diff
@@ -40,12 +41,21 @@ fi
 echo "Done with npm installation test"
 popd
 
+# installs package locally (yarn 1 struggles with registry authentication requirements)
+mkdir okta-auth-js
+cd okta-auth-js
+wget --header="Authorization: Bearer ${NPM_TOKEN}" ${published_tarball} -O ${artifact_version}.tgz
+ls -al
+cd ..
+
 # verify yarn classic install
 mkdir yarn-classic-test
 pushd yarn-classic-test
 yarn init -y
+cp /root/.npmrc .npmrc
```

Comment:
Could this be done as `yarn config` with a local scope?

Reply:
https://stackoverflow.com/a/77067569

> Using yarn config set will not work.